bundles/org.eclipse.passage.lic.base/src/org/eclipse/passage/lic/base/io/Settings.java

```diff
@@ -66,49 +63,26 @@ public Settings(Supplier<Path> base) {
 	}
 
 	public Map<String, Object> get() throws LicensingException {
-		HunterForSettingsFiles hunter = new HunterForSettingsFiles();
-		try {
-			Files.walkFileTree(base.get(), hunter);
+		try (Stream<Path> settingFiles = Files.walk(base.get()).filter(Files::isRegularFile).filter(IS_SETTINGS_FILE)) {
```

Comment:
two concerns here
 - variable name must be of a single, explanatory word
 - init code for it is complex enough to form a method

Reply:
Done.